.github/workflows/upload-cdn.yml

```
@@ -0,0 +1,78 @@
+name: Upload packages to CDN
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Publish packages"]
+    types:
+      - completed
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  CDN_BUCKET: gc-design-system-production-cdn
+  CDN_REGION: ca-central-1
+
+jobs:
+  unable-to-deploy:
+    # If this was triggered by a workflow run ("Publish packages" as noted above) and the workflow run failed
+    if: ${{ github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'failure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Exit and notify Slack if publishing failed and we can't upload to the CDN
+        run: |
+          json='{"blocks":[{"type":"section","text":{"type":"mrkdwn","text":":red: Publishing workflow failed, unable to upload to CDN <https://github.com/cds-snc/gcds-components/actions/workflows/upload-cdn.yml|Upload packages to CDN>"}}]}'
+          curl -X POST -H 'Content-type: application/json' --data "$json" ${{ secrets.SLACK_WEBHOOK_OPS }}
+          exit 1
+
+  upload-to-cdn:
+    if: ${{ github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success' }}
```

This one has to be like so, otherwise it'll block the workflow dispatch triggers:
```suggestion
    if: ${{ github.event_name != 'workflow_run' || github.event.workflow_run.conclusion == 'success' }}
```